modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/MysqlSchemaCodegen.java

```diff
@@ -271,12 +271,11 @@ public void processOpts() {
                 modelDescription = (modelDescription == null || modelDescription.isEmpty()) ? commentExtra : modelDescription + ". " + commentExtra;
             }
 
-            if (modelVendorExtensions.containsKey(CODEGEN_VENDOR_EXTENSION_KEY)) {
+            if (modelVendorExtensions.containsKey(VENDOR_EXTENSION_MYSQL_SCHEMA)) {
```

Comment:
`CODEGEN_VENDOR_EXTENSION_KEY` vs `VENDOR_EXTENSION_MYSQL_SCHEMA`?
What difference does it make? :smile:
Forget it, already done.

Reply:
As a static field, this property has to make sense to consumers not only within this project but for those extending the artifact.

`MysqlSchemaCodegen.CODEGEN_VENDOR_EXTENSION_KEY` doesn't explain what this key is. Although it's technically accurate, you're able to have any number of vendor extensions. The new name removes the redundant `CODEGEN_` prefix and changes `KEY` which provides no details about the public field to `MYSQL_SCHEMA` which represents what the vendor extension is (x-mysql-schema).